packages/stream_chat_localizations/lib/src/stream_chat_localizations_fr.dart

```diff
@@ -0,0 +1,360 @@
+  String get genericErrorText => 'Il y a eu un problème';
+
+  @override
+  String get loadingMessagesError => 'Erreur de chargement des messages';
+
+  @override
+  String resultCountText(int count) => '$count résultats';
+
+  @override
+  String get messageDeletedText => 'Ce message a été supprimé.';
+
+  @override
+  String get messageDeletedLabel => 'Message supprimé';
+
+  @override
+  String get messageReactionsText => 'Réactions aux messages';
+
+  @override
+  String get emptyChatMessagesText => 'Pas encore de chats ici...';
+
+  @override
+  String threadSeparatorText(int replyCount) {
+    if (replyCount == 1) return '1 Réponse';
+    return '$replyCount Replies';
+  }
+
+  @override
+  String get connectedLabel => 'Connecté';
+
+  @override
+  String get disconnectedLabel => 'Déconnecté';
+
+  @override
+  String get reconnectingLabel => 'Reconnexion...';
+
+  @override
+  String get alsoSendAsDirectMessageLabel =>
+      'Envoyer aussi comme message direct';
+
+  @override
+  String get addACommentOrSendLabel => 'Ajouter un commentaire ou envoyer';
+
+  @override
+  String get searchGifLabel => 'Recherche de GIFs';
+
+  @override
+  String get writeAMessageLabel => 'Écrire un message';
+
+  @override
+  String get instantCommandsLabel => 'Commandes instantanées';
+
+  @override
+  String get fileTooLargeAfterCompressionError =>
+      'Le fichier est trop volumineux pour être téléchargé. '
+      'La taille maximale des fichiers est de 20 Mo. '
+      "Nous avons essayé de le compresser, mais ce n'était pas suffisant.";
+
+  @override
+  String get fileTooLargeError =>
+      'Le fichier est trop volumineux pour être téléchargé. La taille limite du fichier est de 20 Mo.';
+
+  @override
+  String emojiMatchingQueryText(String query) =>
+      'Emoji qui correspond à "$query"';
+
+  @override
+  String get addAFileLabel => 'Ajouter un fichier';
+
+  @override
+  String get photoFromCameraLabel => "Photo de l'appareil photo";
+
+  @override
+  String get uploadAFileLabel => 'Transférer un fichier';
+
+  @override
+  String get uploadAPhotoLabel => 'Transférer une photo';
+
+  @override
+  String get uploadAVideoLabel => 'Transférer une vidéo';
+
+  @override
+  String get videoFromCameraLabel => 'Vidéo depuis la camera';
+
+  @override
+  String get okLabel => 'OK';
+
+  @override
+  String get somethingWentWrongLabel => 'Quelque chose a mal tourné';
+
+  @override
+  String get addMoreFilesLabel => "Ajouter d'autres fichiers";
+
+  @override
+  String get enablePhotoAndVideoAccessMessage =>
+      "Veuillez autoriser l'accès à vos photos"
+      '\net vidéos afin de pouvoir les partager avec vos amis.';
+
+  @override
+  String get allowGalleryAccessMessage => "Autoriser l'accès à votre galerie";
+
+  @override
+  String get flagMessageLabel => 'Signaler un message';
+
+  @override
+  String get flagMessageQuestion =>
+      'Voulez-vous envoyer une copie de ce message à une'
+      '\modérateur pour une enquête plus approfondie ?';
```

Comment:
@sachaarbonel is this a mistake?

```suggestion
      '\nmodérateur pour une enquête plus approfondie ?';
```

Reply:
well catch